src/sql/src/plan/query.rs

```diff
@@ -3200,10 +3200,10 @@ fn invent_column_name(
                     _ => unreachable!(),
                 };
 
-                if schema
-                    == &SchemaSpecifier::from(*ecx.qcx.scx.catalog.get_mz_internal_schema_id())
+                // TODO: need to include other unstable schemas too?
```

Comment:
Open question (2)

Reply:
I think we do not need to include unstable here.